_posts/2020-03-29-newpipe-0.19.0-released.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "NewPipe 0.19.0 released: Group your feeds and mute them"
+short: "NewPipe 0.19.0 released"
+date: 2020-03-29 00:00:00 +0100
+categories: [release, pinned]
+author: Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+A new day, a new full release. In today's update, NewPipe is proudly bringing you a mute button and a great way to manage your subscribed channel, on top of the usual smaller improvements and fixes.<!-- more -->
+
+
+## Feed groups
+
+Since moving images say more than my words, please have a look at them:
+
+<video src="/img/folders.webm" class="no-flow img-responsive" width="220" height="140" autoplay loop></video>
+
+For those who prefer to read, this is for you: NewPipe now supports grouping subscriptions into folders. All the recent videos of those channels then appear in one re-loadable feed when you press on the folder. We have to thank [@mauriciocolli](https://github.com/mauriciocolli) for that.
+
+An additional feature squashed into this change is the implementation of a so called _fast-mode_. If you enable it in the <code><nobr><img src="/img/icons/baseline-language-20px.svg" /> Content</nobr></code> settings, those feeds will load much faster from the services which provide a so called _dedicated feed_. The downside of that is that those feeds contain less information, which might result in duration or the live status of videos not shown in the preview. The decision is up to you: Living on the speeding edge and missing information or taking it nice and slow, but knowing **everything**.
+
+## What's New now shows the latest videos first! (Finally!)
+
+Part of the above feature is also the now improved `What's New` sidebar feed. With improved I mean finally a fixed version: The videos are showing in the correct order of the upload, not the first video each channel in a random order. And on top of that, live streams have now a viewer count in those feeds!
```

Comment:
```suggestion
Part of the above feature is also the much improved `What's New` sidebar feed. By improved I mean fixed, finally: The videos now show in the correct order of uploading, whereas the previous version showed the first video of each channel in a random order (roughly in the order you subscribed to them). And on top of that, live streams now have a viewer count in those feeds!
```

Reply:
the header already says finally, kinda doubled...

Reply:
I just moved it from before to after "fixed". Removing it is fine.

Reply:
but now it stands out that much :D